activiti-services/activiti-services-query/activiti-services-query-repo/src/main/java/org/activiti/services/query/app/repository/ProcessInstanceRepository.java

```diff
@@ -17,16 +17,21 @@
 package org.activiti.services.query.app.repository;
 
 import com.querydsl.core.types.dsl.StringPath;
+
 import org.activiti.services.query.model.ProcessInstance;
 import org.activiti.services.query.model.QProcessInstance;
 import org.springframework.data.querydsl.QuerydslPredicateExecutor;
 import org.springframework.data.querydsl.binding.QuerydslBinderCustomizer;
 import org.springframework.data.querydsl.binding.QuerydslBindings;
-import org.springframework.data.repository.CrudRepository;
+import org.springframework.data.rest.core.annotation.Description;
 import org.springframework.data.rest.core.annotation.RepositoryRestResource;
 
-@RepositoryRestResource(exported = false)
-public interface ProcessInstanceRepository extends CrudRepository<ProcessInstance, Long>,
+@RepositoryRestResource(path="/process-instances",
+	collectionResourceDescription = @Description("Collection of process instance resources"),
```

Comment:
Could you check your code styles, please? It seems that you're using tabs stead of whitespaces for indentation.
You can find configuration files [here](https://github.com/Activiti/activiti-build/tree/master/ide-configuration/eclipse-configuration).

Reply:
Done